index.html

```diff
@@ -2,8 +2,12 @@
 <html>
   <head>
     <title>Hangman</title>
+        <link rel="stylesheet" type="text/css" href="hangman.css">
```

Comment:
Do you have a CSS file? if not, I don't think you need to link anything, as it will throw an error